backend/internal/oauth/oauth2/authz/authzhandler_test.go

```diff
@@ -29,24 +29,25 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 
-	"github.com/asgardeo/thunder/internal/oauth/oauth2/authz/constants"
-	"github.com/asgardeo/thunder/internal/oauth/oauth2/authz/model"
+	appprovider "github.com/asgardeo/thunder/internal/application/provider"
+
 	oauth2const "github.com/asgardeo/thunder/internal/oauth/oauth2/constants"
 	oauth2model "github.com/asgardeo/thunder/internal/oauth/oauth2/model"
 	sessionmodel "github.com/asgardeo/thunder/internal/oauth/session/model"
 	"github.com/asgardeo/thunder/internal/system/config"
+	"github.com/asgardeo/thunder/internal/system/jwt"
 )
 
-type AuthorizeHandlerTestSuite struct {
+type authorizeHandlerTestSuite struct {
```

Comment:
```suggestion
type AuthorizeHandlerTestSuite struct {
```

Shall we have this public for consistency?